ansible/roles/common/templates/initialize.sh

```diff
@@ -9,6 +9,8 @@ supervisor_up() {
 }
 until supervisor_up; do echo 'waiting for supervisor ...'; sleep 1; done
 echo 'supervisor up'
+
+/opt/common/fix_supervisor_sock_permissions.sh
```

Comment:
That's a lot of code to fix broken permissions. Let's just run discovery as root for now.